whitepaper/dsnp_whitepaper.tex

```diff
@@ -413,35 +450,34 @@ \subsubsection{Structure}
 \end{figure}
 
 A series of Graph Change Events are depicted in Figure \ref{fig:3}, along with the
-resulting state on the blockchain (notice that the unfollowed friends are no longer
+resulting state on the consensus system (notice that the unfollowed friends are no longer
 present).
 
 \subsubsection{Privacy}
 
 Graph Change Events always have unencrypted Type and Actor values. Graph Change Events of
-type public also have unencrypted \say{Object} and
-\say{Action} values. Graph Change Events of type private encrypt the values for
-\say{Object} and Action. This means that, despite the Graph Change Event being published
-to the blockchain, a key is required to interpret the encrypted values. A user generates a
-symmetric encryption key for encrypting the private Graph Change Events data. The user can
-store this encryption key in their wallet or on chain (see the \say{Key Storage} section).
-While private Graph Change Events do not allow outside observers to read who is being
-followed, it is still possible to see that the Social Identity expressed in the Actor
-field is actively publishing a particular type of event, which may still convey some level
-of valuable metadata information to a third party.
+type public also have unencrypted \say{Object} and \say{Action} values. Graph Change Events
+of type private encrypt the values for \say{Object} and Action. This means that, despite the
+Graph Change Event being published to the consensus system, a key is required to interpret
+the encrypted values. A user generates a symmetric encryption key for encrypting the private
+Graph Change Events data. The user can store this encryption key in their wallet or on chain
```

Comment:
Does this need a footnote that mentions that the encryption system is updated in the spec?